### PR TITLE
dotabuff.com

### PR DESCRIPTION
**Pull request by galeksandrp**
Issue author: @galeksandrp

### CODE REVIEW

File: src/chrome/content/rules/dotabuff.com.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+www.webmail.dotabuff.com ¹
+www.www.dotabuff.com ¹
+2.www.dotabuff.com ¹
+bg.www.dotabuff.com ¹
+en.www.dotabuff.com ¹
+es.www.dotabuff.com ¹
+w.gowww.www.dotabuff.com ¹
+ru.www.dotabuff.com ¹
+zh.www.dotabuff.com ¹
+www.zh.dotabuff.com ¹
```
Comment (J0WI) [inline src/chrome/content/rules/dotabuff.com.xml]: They have a wildcard DNS entry

Comment (J0WI) [inline src/chrome/content/rules/dotabuff.com.xml]: And also a wildcard cert

Comment (galeksandrp) [inline src/chrome/content/rules/dotabuff.com.xml]: https://github.com/EFForg/https-everywhere/pull/4936#issuecomment-222307982
https://github.com/EFForg/https-everywhere/pull/4931#issuecomment-222306666

Comment (J0WI) [inline src/chrome/content/rules/dotabuff.com.xml]: I would prefer a comment about this behavior. 4th+ level domain will mismatch anyway.
The target list can be reduced to those hosts that are really in use.
A list of hosts makes no sense here.

Comment (galeksandrp) [inline src/chrome/content/rules/dotabuff.com.xml]: > 4th+ level domain will mismatch anyway
Why would you sure about that? Even on same IP SNI can differ certificates for different domains.

**Pull request closed without merging**